src/app/+search-page/search-page.module.ts

```diff
@@ -46,13 +45,12 @@ const effects = [
     SearchResultsComponent,
     SearchSidebarComponent,
     SearchSettingsComponent,
-    ItemSearchResultListElementComponent,
-    CollectionSearchResultListElementComponent,
-    CommunitySearchResultListElementComponent,
+    // ItemSearchResultListElementComponent,
+    // CollectionSearchResultListElementComponent,
+    // CommunitySearchResultListElementComponent,
```

Comment:
Is there a reason to comment these out instead of removing them?

Reply:
Removed commented out declarations